wire-ios-request-strategy/Sources/Notifications/Push Notifications/Notification Types/LocalNotificationContentType.swift

```diff
@@ -80,6 +80,7 @@ public enum LocalNotificationContentType: Equatable {
             self = .knock
 
         case .image:
+            // NOTE: is this v2 asset ?
```

Comment:
Does this need to become solved first?

Reply:
no, it's just an annotation, at some point we might need to clean